src/main/resources/templates/fragments/userBar.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <title>User Bar</title>
+    </head>
+    <body>
+        <div th:fragment="userBar" class="js-toggle-nav" id="global-nav">
+            <a href="#navigation" id="signed-in-user-href" class="js-header-toggle" text="jsmith@testcompany.co.uk:"></a>
```

Comment:
hardcoded text?

Reply:
Just a placeholder, will be replaced with model attributes in due course :)